kuma/wiki/management/commands/submit_deleted_documents.py

```diff
@@ -0,0 +1,96 @@
+from datetime import timedelta
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.db import models
+from django.utils import timezone
+
+from kuma.spam.akismet import Akismet, AkismetError
+from kuma.wiki.forms import revision_akismet_parameters
+from kuma.wiki.models import (Document, DocumentDeletionLog,
+                              RevisionAkismetSubmission)
+
+
+class Command(BaseCommand):
+    help = "Seed Akismet spam with documents that were deleted because they were spam"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('username', type=str)
+        parser.add_argument('-n', '--dry-run',
+                            action='store_true', dest='dry_run', default=False,
+                            help="Do everything except actually submit.")
+        parser.add_argument('-d', '--days',
+                            action='store_true', dest='days', default=365,
```

Comment:
I think a better default should relate to how often we might run this. If we're thinking we might run this once a week via a cron job, then we'd want this to default to 7--not 365.